earth_enterprise/Upgrade.md

```diff
@@ -1,14 +1,19 @@
-# Upgrading Earth Enterprise Fusion and Server to 5.2.1
+# Upgrading Earth Enterprise Fusion and Server to version 5.2.1
 
-Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.1.3 and 5.2.0 versions.
+Upgrading to GEE 5.2.1 is supported from Earth Enterprise 5.2.0 version using the following steps:
 
-## Upgrading GEE 5.2.0 to 5.2.1
+1. Do NOT uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your PostgreSQL databases are backed up and upgraded correctly to the new PostgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server). This can be achieved by running the scripts to install fusion and earth server:
 
-1. Do not uninstall GEE 5.2.0. We recommend that you upgrade GEE 5.2.0 by simply installing GEE 5.2.1. Installing GEE 5.2.1 on top of GEE 5.2.0 will ensure that your postgreSQL databases are backed up and upgraded correctly to the new postgreSQL version used by GEE 5.2.1. [Install Fusion or Earth Server](https://github.com/google/earthenterprise/wiki/Install-Fusion-or-Earth-Server)
+cd earth_enterprise/src/installer
```

Comment:
Style: You could put these commands in a code block. (Add a line of ` ```bash ` before, and a line of ` ``` ` after.)

Reply:
sure